frontend/src/pages/analytics/path/AnalyticsPath.tsx

```diff
@@ -586,385 +471,374 @@ const AnalyticsPath: React.FC = () => {
-                          setSegmentationOptionData((prev) => {
-                            const dataObj = cloneDeep(prev);
-                            dataObj.data[index].conditionOption = category;
-                            if (
-                              category?.valueType === MetadataValueType.STRING
-                            ) {
-                              dataObj.data[index].conditionValue = [];
-                            } else {
-                              dataObj.data[index].conditionValue = '';
-                            }
-                            return dataObj;
-                          });
-                        }}
-                        changeConditionOperator={(index, operator) => {
-                          setSegmentationOptionData((prev) => {
-                            const dataObj = cloneDeep(prev);
-                            dataObj.data[index].conditionOperator = operator;
-                            return dataObj;
-                          });
-                        }}
-                        changeConditionValue={(index, value) => {
-                          setSegmentationOptionData((prev) => {
-                            const dataObj = cloneDeep(prev);
-                            dataObj.data[index].conditionValue = value;
-                            return dataObj;
-                          });
-                        }}
-                        changeCurRelationShip={(relation) => {
-                          setSegmentationOptionData((prev) => {
-                            const dataObj = cloneDeep(prev);
-                            dataObj.conditionRelationShip = relation;
-                            return dataObj;
-                          });
+                    <div className="cs-analytics-session-window-unit">
+                      <Select
+                        selectedOption={selectedWindowUnit}
+                        options={windowUnitOptions}
+                        onChange={(event) => {
+                          setSelectedWindowUnit(event.detail.selectedOption);
                         }}
                       />
                     </div>
-                  </SpaceBetween>
-                </ColumnLayout>
-                <br />
-                <Button
-                  variant="primary"
-                  iconName="search"
-                  onClick={clickPreview}
-                  loading={loadingData}
+                  </>
+                ) : null}
+              </div>
+            </SpaceBetween>
+            <SpaceBetween direction="vertical" size="xs">
+              <Box variant="awsui-key-label">
+                {t('analytics:labels.nodeType')}{' '}
+                <Popover
+                  triggerType="custom"
+                  size="small"
+                  content="This instance contains insufficient memory. Stop the instance, choose a different instance type with more memory, and restart it."
```

Comment:
What is this?

Reply:
translate

Reply:
These locations are reserved for prompt information, and @luorobin-a2z  can edit at[ i18n files](https://github.com/awslabs/clickstream-analytics-on-aws/pull/241/files#diff-e54d97e6bd16f002cbf98607fce7aa874d5d0d7c011011abce5c9a00cf4dae22R240).

Reply:
Will update them in this PR or subsequent one?

Reply:
Edit prompt information in other PR.